### PR TITLE
Bekmurod

### PR DESCRIPTION
**Pull request by BekjoN2003**
Lib version ga TokenResponse va UserDto class lari qo'shildi

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -23,9 +23,12 @@ repositories {
 }
 
 dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-security:2.4.0'
```
Comment (Axmadjanovich) [inline build.gradle]: Agar dependency ishlatilmasa, qo'shish kerak emas

**Review by Axmadjanovich — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 2]
```diff
@@ -23,9 +23,12 @@ repositories {
...
+	implementation 'io.jsonwebtoken:jjwt-jackson:0.11.2'
+	implementation 'org.projectlombok:lombok:1.18.26'
```
Comment (Axmadjanovich) [inline build.gradle:28]: lombok oldin bor edi

File: build.gradle [hunk 3]
```diff
@@ -23,9 +23,12 @@ repositories {
 }
 
 dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-security:2.4.0'
+	implementation 'io.jsonwebtoken:jjwt-jackson:0.11.2'
```
Comment (Axmadjanovich) [inline build.gradle:27]: JWT kerak bo'lmasa olib tashlash kerak

**Review by Axmadjanovich — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (BekjoN2003) [reply build.gradle]: muammo hal qilindi

File: build.gradle (hunk 3 above)
Comment (BekjoN2003) [reply build.gradle:27]: bu dependecy UserDetails ni qo'shish uchun kerak

**Pull request merged**